src/styles/BurgerButton.scss

```
@@ -0,0 +1,68 @@
+@import './Constants.scss';
+
+.burger-button {
+    display: inline-block;
+    cursor: pointer;
+    position: fixed;
+    margin: 0px;
+    width: 30px;
+    right: 53px;
+    z-index: 3;
+    background-color: $secondaryDark;
+  }
+  
+  @mixin bar {
+    width: 35px;
+    height: 5px;
+    background-color: $primaryText;
+    margin: 6px 0;
+    transition: 0.4s;
+  }
+
+  .opened-bar-1, .opened-bar-2, .opened-bar-1 {
+    @include bar; 
+  }
+  
+  /* Rotate first bar */
+  .closed-bar-1 {
+    @include bar;
+    -webkit-transform: rotate(-45deg) translate(-9px, 6px) ;
+    transform: rotate(-45deg) translate(-9px, 6px) ;
+  }
+  
+  /* Fade out the second bar */
+  .closed-bar-2 {
+    @include bar;
+    opacity: 0;
+  }
+  
+  /* Rotate last bar */
+  .closed-bar-3 {
+    @include bar;
+    -webkit-transform: rotate(45deg) translate(-8px, -8px) ;
+    transform: rotate(45deg) translate(-8px, -8px) ;
+  }
+
+  @media(min-width: $tabletScreenSize) {
```

I would probably do this instead, we don't need a variable for a size that is 1px larger than another variable. Also if $mobileScreenSize is the max screen size of a mobile, I would expect $tabletScreenSize to be the max screen size of the tablet, which isn't what this represents here
```suggestion
  @media(min-width: $mobileScreenSize + 1px) {
```